jsonschema-core.xml

```diff
@@ -135,13 +135,66 @@
                 data in several ways.
             </t>
             <t>
-                JSON Schema uses keywords to assert constraints on JSON instances or annotate those
-                instances with additional information.  Additional keywords are used to apply
-                assertions and annotations to more complex JSON data structures, or based on
-                some sort of condition.
+                A JSON Schema document is comprised of a set of keywords,
+                which are specified as properties in a JSON object.
+                The name of the keyword is used as the property name,
+                and any arguments to the keyword are provided as the value.
+                Keywords can be classified by their functions.
+                The most important classes are validation, annotation, and applicator keywords.
             </t>
             <t>
-                To facilitate re-use, keywords can be organized into vocabularies. A vocabulary
+                "Validation keywords" form the basis of JSON Schema.
+                Given a schema and an instance, the instance is "valid" if all of the keywords in
+                the schema are valid against the instance.
+                Validation keywords may be used to describe a set of JSON documents.
+                Without any validation keywords, the set of all valid instances is the set of all valid JSON documents.
```

Comment:
Do people or specifications use the term "JSON document" to refer to non-object typed JSON? If not this is a bit imprecise.

Reply:
Maybe a different term should be used because JSON doesn't use the term "JSON document."

But "document" has an Internet meaning, so "JSON document" should imply "document with media type `application/json`"

Reply:
Also, older versions of JSON required that documents be an array or an object... this restriction has since been lifted, a document can be simply `true` or `null` or a number.

Reply:
Right, as I say I just don't know that people use the word document to refer to those. Values maybe. But I could be wrong.